libs/web-components/src/components/calendar/Calendar.svelte

```diff
@@ -307,14 +308,16 @@
         maxheight="240px"
         value={_calendarDate?.month}
         on:_change={setMonth}
+        {version}
+        size={version === "2" ? "compact" : "default"}
       >
         {#each _months as month, i}
           <goa-dropdown-item value={i+1+""} label={month} />
         {/each}
       </goa-dropdown>
     </goa-form-item>
 
-    <goa-form-item label="Year" mt="0">
+    <goa-form-item label="Year" mt="0" {version}>
```

Comment:
FormItem labelsize should match compact dropdowns in V2.


```suggestion
    <goa-form-item label="Year" mt="0" {version} labelsize={version === "2" ? "compact" : "regular"}>
```